File: 4-WP-web-programming/Exam/Exam Samples/examjsp/src/main/java/ubb/exam/model/ItemUser.java
package ubb.exam.model;

public class ItemUser extends BaseEntity {
    public String username;
    public Integer itemCount;
    public Integer totalValue;

    public ItemUser() {}

    public ItemUser(String username, Integer itemCount, Integer totalValue) {
        this.username = username;
        this.itemCount = itemCount;
        this.totalValue = totalValue;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public Integer getItemCount() {
        return itemCount;
    }

    public void setItemCount(Integer itemCount) {
        this.itemCount = itemCount;
    }

    public Integer getTotalValue() {
        return totalValue;
    }

    public void setTotalValue(Integer totalValue) {
        this.totalValue = totalValue;
    }
}

File: 4-WP-web-programming/Exam/Exam Samples/examphp/itemuser.php
<?php
class ItemUser implements JSONSerializable {
    private $item_id;
    private $username;
    private $itemCount;
    private $totalValue;

    public function __construct($item_id, $username, $itemCount, $totalValue) {
        $this->item_id = $item_id;
        $this->username = $username;
        $this->itemCount = $itemCount;
        $this->totalValue = $totalValue;
    }

    public function getId() {
        return $this->item_id;
    }

    public function getUsername() {
        return $this->username;
    }

    public function getItemCount() {
        return $this->itemCount;
    }

    public function getTotalValue() {
        return $this->totalValue;
    }

    public function jsonSerialize() {
        return get_object_vars($this);
    }

}
?>